alacritty_terminal/src/config/mod.rs

```diff
@@ -215,6 +215,22 @@ impl From<CursorBlinking> for bool {
     }
 }
 
+#[derive(ConfigDeserialize, Default, Debug, Clone, Copy, PartialEq, Eq)]
+pub enum OptionAsAlt {
+    /// The left `Option` key is treated as `Alt`.
+    OnlyLeft,
+
+    /// The right `Option` key is treated as `Alt`.
+    OnlyRight,
+
+    /// Both `Option` keys are treated as `Alt`.
+    Both,
+
+    /// No special handling is applied for `Option` key.
+    #[default]
+    None,
+}
+
```

Comment:
This is only used in alacritty, the type has no use in `alacritty_terminal`, because it doesn't do anything related to keyboard input, just simply move it into `alacritty`.

Reply:
Ha, what a great point. Thanks.